### PR TITLE
Clinical visits filter by clinical case

### PR DESCRIPTION
**Pull request by marius2301**
Fixes #

### CODE REVIEW

**Review by MartinWahnschaffe — CHANGES_REQUESTED** [review]

**Comment by MartinWahnschaffe** [comment]
@marius2301 Looks good. Let's wait until the release is done (likely Wednesday) before merging.

File: sormas-rest/src/main/java/de/symeda/sormas/rest/ClinicalVisitResource.java
Comment (marius2301) [reply sormas-rest/src/main/java/de/symeda/sormas/rest/ClinicalVisitResource.java]: Added!

**Pull request merged**
